### PR TITLE
Add discussion template

### LINKED ISSUE #169: Discussion template
## Problem

Discussions have historically been spread across GitHub discussions and issues. We'd like to move them to issues only, and make it easy for anyone to create a new discussion issue.

## Description

We should create a synced issue template across all of our repos for discussions. Some fields:

- Due date
- Assigned reviewers
- Automatically apply the `talk: discussion` label

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by zackkrida**
## Fixes

Fixes #169 by @zackkrida 

## Description

Adds a simple issue template to start discussions. Please adjust as you see fit!

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by zackkrida** [comment]
@WordPress/openverse-developers could someone `just lint` and commit to this PR for me? I'm having a local pyenv issue I'll need help on sometime and can't run the check myself 😊

File: .github/ISSUE_TEMPLATE/discussion.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+name: Discussion
+about: Encourage conversation among contributors
+labels: "💬 talk: discussion"
+title: "Discuss <Replace this with actual title>"
```
Comment (dhruvkb) [inline .github/ISSUE_TEMPLATE/discussion.md:5]:
```suggestion
title: "<Replace this with actual title>"
```

The "Discuss" is implied in the label.

**Review by dhruvkb — APPROVED** [review]
LGTM!

**Review by stacimc — APPROVED** [review]
Looks good!

**Pull request merged**